org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/FindWindow.java

```diff
@@ -1242,10 +1243,12 @@ private void setValues(Listbox listColumn, Listbox listOperator, String[] fields
             	{
             		listOperator.setSelectedItem(li);
             		selected = true;
-            		break;
```

Comment:
same here, why remove the break ?

Reply:
The list of operators was being filled incomplete.